### ISSUE TITLE
Inconsistent indentation in file icon themes

### ISSUE THREAD

**Issue by nilsocket** [microsoft/vscode#125644]
Does this issue occur when all extensions are disabled?
Yes

- VS Code Version: 1.56.2, `054a9295330880ed74ceaedda236253b4f39a335`, x64
- OS Version: Linux 5.10.36-2-MANJARO

Seti - File Icon Theme:
![vscode-seti](https://user-images.githubusercontent.com/29815708/120973782-feda4b80-c78c-11eb-8b5c-db8d00e8aafc.png)

Minimal - File Icon Theme:
![vscode-minimal](https://user-images.githubusercontent.com/29815708/120973839-131e4880-c78d-11eb-9a7a-6787eebe0630.png)

None - File Icon Theme:
![vscode-none](https://user-images.githubusercontent.com/29815708/120973875-1fa2a100-c78d-11eb-9c56-f25e5fb212ad.png)

In both `Minimal` and `None` it's hard to follow to which folder does `.gitignore` and `Readme.md` belongs to, they actually belong to parent folder of `Mathematics`, but in-case of `minimal` and `none` they look like they belong to `Mathematics` folder.

Thank you.

**Comment by aeschli** [comment]
The idea is that the label is always aligned. It has been like that for many years.

Do you use a custom font?

Note we have the indent guide that help as well:
Setting is `workbench.tree.renderIndentGuides`

![image](https://user-images.githubusercontent.com/6461412/121033281-589e3e00-c7ac-11eb-9559-62c1c92ddf4b.png)

**Comment by nilsocket** [comment]
> Do you use a custom font?

No, I don't use any custom font.

> The idea is that the label is always aligned.

For `Minimal`:
- 2 icons for folder,  `>` and folder `Icon`
- One for File, file `Icon`

For `None`:
- 1 icon for folder, `>`
- None for File.

Indirectly, 1 extra Icon is being used for folder, which is causing these problems.

Possible Solutions:

1. Using one Icon only (like `seti`)
    - For `Minimal`, Remove folder `Icon`.
    - For `None`, I'm not sure what would be the solution (Maybe moving arrow to left, but that would be just be yet another inconsistency).

2. Using two Icons (Screenshot from KDE - Dolphin File Manager):
    - For `Minimal`:
        - Preview Icon size set to `16px`
![dolphin-16-Pixels](https://user-images.githubusercontent.com/29815708/121046528-a5493f80-c7d3-11eb-870b-757e6eb69f4f.png)
        - Preview Icon size set to `96px`
![dolphin-96-Pixels](https://user-images.githubusercontent.com/29815708/121046643-abd7b700-c7d3-11eb-9f3d-f8754066bd36.png)
    - For `None`, `>` for folder, `-` for file, forming tree structure like above without any icons.

On a sideNote:
Rather than having a fixed pixel size for indentation, wouldn't it be more appropriate having icon width as indent space.

Thank you.

**Comment by aeschli** [comment]
We allow different looks for icon themes.
- No icons (e.g None)
- Only file icons (e.g. VS Code Seti)
- Folder and File Icons (e.g. Minimal)
- Folder and File Icons without arrows (e.g. the Material theme has an option to enabled that: "material-icon-theme.hidesExplorerArrows": true)

This is made consistent by 
- labels are always aligned. 
- Icons are fixed size. 
- Indent size can be customized: `workbench.tree.indent`

Whether icons are aligned depends very much on the icon theme.

Additionally we have the indent guides to clarify the hierarchy level.

Thanks for your input and suggestions. We currently have no plans to change the built-in icon themes or the way icons are rendered as this would impact a lot of users and well established icon themes.

I suggest you to play with the indent setting and try the various icon themes on the marketplace to see which suites you the best
